css-paint-api/Overview.bs

```diff
@@ -216,11 +255,14 @@ called, the user agent <em>must</em> run the following steps:
         - <a>paint constructor valid flag</a> being true
 
         - <a>paint context alpha flag</a> being |alpha|.
-    
-    14. Add the key-value pair (|name| - |inputProperties|) to the <a>paint name to input properties
+
+    19. Add the key-value pair (|name| - |inputProperties|) to the <a>paint name to input properties
```

Comment:
Use the Infra algos for manipulating a map? <https://infra.spec.whatwg.org/#map-set>